WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -62,15 +70,24 @@ private async Task<bool> ConfirmConnectionAsync(CancellationToken cancellationTo
 
 			var amountsToRequest = new[] { effectiveAmount };
 
-			return await ArenaClient
+			var response = await ArenaClient
```

Comment:
Once this is called, calling `ConfirmConnectionAsync` again is invalid.

Maybe it's obvious now, but I think it adds little complexity and pays off by writing down implicit assumptions if Alice also tracks its state, i.e. in flight request starting here, and then completed below, with success based on the `response.Value`

Reply:
In case the `ConfirmConnectionAsync` is called again that would mean we have a bug somewhere else and we have to fix it.

Reply:
yep. is it worth it to add code to detect such a bug if it ever comes up? in this case i would say maybe yes, since that can be done easily, this is because Alice already needs to track most of the state required to detect such misuse when it manages the input registration response data that connection confirmation relies on. i don't feel strongly and i can also make an argument that this is just adding overly defensive code, and even with low complexity it's not worth it, so feel free to ignore.

Reply:
If it is just a boolean I would add it.